src/service/viewer-impl.js

```diff
@@ -1005,7 +1005,7 @@ export class Viewer {
       return Promise.reject(getChannelError());
     }
     return this.messagingReadyPromise_.then(() => {
-      return this.sendMessageUnreliable_(eventType, data, awaitResponse);
+      return this.messageDeliverer_(eventType, data, awaitResponse);
```

Comment:
Imagine we add a 4th param to `messageDeliverer` function. Now we have to add that to 3 spots instead of just `#sendMessageUnreliable`.